laa_court_data_api_app/routers/defendants.py

```diff
@@ -33,6 +34,14 @@ async def get_defendants(urn: Optional[str] = None,
         logging.info(f"Defendants_Get_Urn_{urn}")
         cda_response = await client.get("/api/internal/v2/prosecution_cases",
                                         params={"filter[prosecution_case_reference]": urn})
+    elif asn:
+        logging.info(f"Defendants_Get_Urn_{asn}")
+        cda_response = await client.get("/api/internal/v2/prosecution_cases",
+                                        params={"filter[arrest_summons_number]": asn})
+    elif nino:
+        logging.info(f"Defendants_Get_Urn_{nino}")
+        cda_response = await client.get("/api/internal/v2/prosecution_cases",
+                                        params={"filter[national_insurance_number]": nino})
```

Comment:
Logs are a little bit misleading as they all indicate Get_Urn even though we are getting via ASN